### PR TITLE
Fix parsing of ISO 8601 dates

### PR DESCRIPTION
**Pull request by mdiep**
`DD` is day of year; `dd` is day of month.

### CODE REVIEW

File: SquirrelTests/SQRLUpdateSpec.m [hunk 1]
```diff
@@ -45,12 +45,12 @@
...
 it(@"should parse Central style dates", ^{
 	SQRLUpdate *update = [MTLJSONAdapter modelOfClass:SQRLUpdate.class fromJSONDictionary:@{ @"url": @"http://example.com/update", @"pub_date": @"Tue Sep 17 10:24:27 -0700 2013" } error:NULL];
-	expect(update.releaseDate).notTo(beNil());
+	expect(update.releaseDate).to(equal([NSDate dateWithTimeIntervalSince1970:1379438667]));
 });
 
 it(@"should parse ISO 8601 dates", ^{
 	SQRLUpdate *update = [MTLJSONAdapter modelOfClass:SQRLUpdate.class fromJSONDictionary:@{ @"url": @"http://example.com/update", @"pub_date": @"2013-09-18T13:17:07+01:00" } error:NULL];
-	expect(update.releaseDate).notTo(beNil());
+	expect(update.releaseDate).to(equal([NSDate dateWithTimeIntervalSince1970:1379506627]));
```
Comment (mdiep) [inline SquirrelTests/SQRLUpdateSpec.m]: This test failed before 09a3b74.

**Comment by joshaber** [comment]
Amazing. Nice catch :+1:

**Comment by mdiep** [comment]
Always test for equality, kids.

**Comment by alanjrogers** [comment]
:clap:

**Pull request merged**